conjur/client.py

```diff
@@ -171,17 +171,17 @@ def whoami(self):
         return self._api.whoami()
 
     # Constraints remain an optional parameter for backwards compatibility in the SDK
-    def list(self, list_constraints=None):
+    def list(self, *list_constraints):
```

Comment:
why with *?
if its optional why not list_constraints=None

Reply:
you are right. changing back. i confused myself hehe